subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/InstantExecutionBuildOptionsIntegrationTest.groovy

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.instantexecution
+
+import spock.lang.Unroll
+
+class InstantExecutionBuildOptionsIntegrationTest extends AbstractInstantExecutionIntegrationTest {
```

Comment:
We should add some int test coverage in `model-core` to live with the feature, some of which would replace tests in `instant-execution`. We can do this in another PR, however.